### PR TITLE
Canvas component done

### PR DESCRIPTION
**Pull request by sashkivna**

### CODE REVIEW

File: scripts/main/designer/designer.directive.js [hunk 1]
```diff
@@ -15,7 +15,9 @@ angular.module('designer.module')
 
                 $scope.addComponentToCanvas = function (component) {
                     $scope.canvasComponents.push(component);
-                }
+                };
+
```
Comment (r0ss0neri) [inline scripts/main/designer/designer.directive.js:19]: remove blank lines

File: scripts/styles/styles.css [hunk 2]
```diff
@@ -36,4 +36,12 @@ body {
 
 .selected-component {
     border: 2px solid palevioletred;
+}
+
+.type {
```
Comment (r0ss0neri) [inline scripts/styles/styles.css]: use more weighty selector

    foo-component {
       ...
    }

File: scripts/styles/styles.css [hunk 3]
```diff
@@ -36,4 +36,12 @@ body {
 
 .selected-component {
```
Comment (r0ss0neri) [inline scripts/styles/styles.css]: stensil {
        .selected-component {
            ...
        }
    }

**Review by r0ss0neri — CHANGES_REQUESTED** [review]

File: scripts/main/designer/designer.directive.js (hunk 1 above)
Comment (sashkivna) [reply scripts/main/designer/designer.directive.js:19]: done

File: scripts/styles/styles.css (hunk 2 above)
Comment (sashkivna) [reply scripts/styles/styles.css]: done

File: scripts/main/designer/components/stencil/stencil.template.html [hunk 4]
```diff
@@ -1,6 +1,6 @@
 <div ng-repeat="component in components">
     <div class="component"
-         ng-class="{'selected-component': isComponentSelected(component)}">
+         ng-class="{'foo-component': isComponentSelected(component)}">
```
Comment (r0ss0neri) [inline scripts/main/designer/components/stencil/stencil.template.html]: leave `selected-component` because stencil don't know which component comes

**Review by r0ss0neri — APPROVED** [review]

**Pull request merged**